### PR TITLE
Add auto-empty command to examples

### PR DESCRIPTION
**Pull request by capstan1**
it's a great feature, needs to be in the docs.
Also think the switches created should be explained somewhere. Maybe a new subheader "settings"?

### CODE REVIEW

File: docs/integrations/home-assistant/examples/commands.md [hunk 1]
```diff
@@ -57,3 +57,16 @@ data:
     3. At the end of the job you can find your coordinates under `data->content`
 
     Also the coordinates will be logged on `debug`. After activating debug logs, you can search your logs for `Last custom area values (x1,y1,x2,y2):` to get the coordinates.
+
+## Empty dustbin to aute-empty station
```
Comment (edenhaus) [inline docs/integrations/home-assistant/examples/commands.md:61]:
```suggestion
## Empty dustbin to auto-empty station
```

**Review by edenhaus — CHANGES_REQUESTED** [review]
thanks for your contribution.
Can you please add also the command `setAutoEmpty` in the [commands section](https://deebot.readthedocs.io/advanced/data_types/json/commands/).
this help to add support for this command in the library

**Comment by capstan1** [comment]
I'm sorry, I have no idea what all the REST stuff is about. So I can't really help you there. 😞

**Comment by edenhaus** [comment]
@Edo78 Have you the technical skills to provide more information about the command?

**Comment by Edo78** [comment]
Sorry, I copy&pasted something from an issue of the previous repo. My skill on HA are pretty low (all my automations are in node red)

**Comment by edenhaus** [comment]
Thanks @capstan1

**Pull request merged**